packages/strapi/lib/Strapi.js

```diff
@@ -318,9 +318,15 @@ class Strapi {
       configuration: this.config.get('currentEnvironment.server.webhooks', {}),
     });
 
+    const ensureDefaultConnection = model =>
+      _.merge(
+        { connection: this.config.get('currentEnvironment.database.defaultConnection') },
```

Comment:
can you instead just set the field in the model ?

Reply:
8a228b3195505acba0ab231c206506c8844b515c